README.md

```diff
@@ -1,51 +1,24 @@
 # Docker image for FACT_core
 The docker image provides an installation of all fact components (db, frontend,
 backend). Currently, you have to build it yourself, but we plan to provide a
-prebuilt image via Dockerhub. For instructions see [Advanced usage](#advanced-usage).
+prebuilt image via Dockerhub. For instructions see [Usage](#usage).
 
 Because FACT uses docker itself, the docker socket from the host will be
 passed to the container. Please make sure that your user is a member of the
 `docker` group.
 
-## Most simple usage
-If you just want to run FACT and not worry about anything we've got your back.
-Be aware that all state of FACT is only in the container so if you pull a new
-FACT version everything (most importantly your database) will be lost.
-
-> Note that the image is currently not uploaded so you have to
-[build](#advanced-usage) it yourself
-
-Simply run
-```bash
-# Pull docker containers needed on the host
-# This is just needed the first time you use FACT
-docker run \
-	--rm \
-	-it \
-	--group-add $(getent group docker | cut -d: -f3) \
-	-v /var/run/docker.sock:/var/run/docker.sock \
-	fkiecad/fact pull-containers
-
-# The docker daemon will write some things in there
-mkdir /tmp/fact-docker-tmp && chgrp docker /tmp/fact-docker-tmp
-
-# Start fact
-docker run \
-	-it \
-	--name fact \
-	--group-add $(getent group docker | cut -d: -f3) \
-	-v /var/run/docker.sock:/var/run/docker.sock \
-	-v /tmp/fact-docker-tmp:/tmp/fact-docker-tmp \
-	-p 5000:5000 \
-	fkiecad/fact start
-
-```
-
-## Advanced usage
+## Usage
 For more advanced usage we provide a Python script to cover most usecases.
 If your usecase is not supported feel free to open a PR or hack it up privately.
 
-Run `./start.py --help` to get help abuot the usage of the script.
+Probably the most imporant command is `./start.py build`.
+After that you might run:
+```sh
+$ mkdir ~/fact_mongo && mkdir ~/fact_data
+$ ./start.py run --fact-wt-mongodb-path ~/fact_mongo --fact-fw-data-path ~/fact_data
```

Comment:
> start.py run: error: the following arguments are required: --wt-mongodb-path, --fw-data-path

The parameter names seem to be different? (`fact-` would be kinda redundant anyway)

Reply:
Whoops, I copy pasted the wrong thing.